src/ui/components/FileListItem/index.jsx

```diff
@@ -30,8 +30,8 @@ const FileListItem = ({
               title="Copy link to your clipboard"
               role="button"
               onClick={() => {
-                  clipboard.writeText(url, 'selection');
-                  ipcRenderer.send(IPC_EVENT_NOTIFICATION, 'Link copied to your clipboard');
+                  clipboard.writeText(url);
+                  ipcRenderer.send(IPC_EVENT_NOTIFICATION, 'Link copied to your clipboard: ' + url);
```

Comment:
Hmm, I wonder if this might be hard to read since the links are very long. Pulling this down to try it out.

Also, minor nit, but you could use the ES2015 string interpolation which tends to read a bit nicer.

```js
ipcRenderer.send(IPC_EVENT_NOTIFICATION, `Link copied to your clipboard: ${url}`);
```

Reply:
gonna skip that. I reverted the output message.